src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,77 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Paginations {
+  total: number
+  currentPage: number
+  perPage: number
+  onPageChange: (page: number) => void
+}
+
+export const Pagination = ({
+  total,
+  currentPage,
+  perPage,
+  onPageChange,
+}: Paginations) => {
+  function onPrevPage() {
+    return currentPage !== 1 && onPageChange(currentPage - 1);
+  }
+
+  function onNextPage() {
+    return currentPage !== Math.ceil(total / (perPage))
+      && onPageChange(currentPage + 1);
+  }
+
+  return (
+    <ul className="pagination">
+      <li className={classNames('page-item',
+        { disabled: currentPage === 1 })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
+          onClick={onPrevPage}
+        >
+          «
+        </a>
+      </li>
+
+      {getNumbers(1, Math.ceil(total / (perPage)))
+        .map(item => (
+          <li
+            className={classNames('page-item',
+              { active: currentPage === item })}
+            key={1}
```

Comment:
```suggestion
            key={item}
```
key must be unique for all items